parent/pom.xml

```diff
@@ -82,7 +82,7 @@
         <jaxb.impl.version>2.1.14</jaxb.impl.version>
         <javassist.version>3.20.0-GA</javassist.version>
         <javax.cache.bundle.version>1.0.0_1</javax.cache.bundle.version>
-        <javax.cache.tck.version>1.0.1</javax.cache.tck.version>
+        <javax.cache.tck.version>1.1.0</javax.cache.tck.version>
```

Comment:
Why the version has changed? Why not `1.1.1` if it's necessary?

Reply:
This version is already used by TeamCity (the value is specified via an environment variable) in accordance with https://issues.apache.org/jira/browse/IGNITE-8548. I just aligned the version with the value which is actually used.